packages/pigz/pigz.spec

```
@@ -0,0 +1,39 @@
+%global debug_package %{nil}
+
+Name: %{_cross_os}pigz
+Version: 2.7
+Release: 1%{?dist}
+Summary: pigz is a parallel implementation of gzip which utilizes multiple cores
+License: Zlib AND Apache-2.0
+URL: http://www.zlib.net/pigz
+Source0: pigz-%{version}.tar.gz
+BuildRequires: %{_cross_os}glibc-devel
+BuildRequires: %{_cross_os}libz-devel
+
+%description
+%{summary}.
+
+%prep
+%autosetup -n pigz-%{version} -p1
+
+%global set_env \
+%set_cross_build_flags \\\
+export CC=%{_cross_target}-gcc \\\
+export PREFIX=%{_cross_prefix} \\\
+export DESTDIR=%{buildroot}%{_cross_rootdir} \\\
+%{nil}
+
+%build
+%set_env
+%make_build
```

The way `CC`, `CFLAGS` and `LDFLAGS` are defined in the Makefile means that they need to be overridden like this, and can't just be set in the environment:
```suggestion
%make_build CC="${CC}" CFLAGS="${CFLAGS}" LDFLAGS="${LDFLAGS}"
```

This will fix the missing debug symbols error also.